src/main/java/io/jenkins/infra/repository_permissions_updater/hosting/Hoster.java

```diff
@@ -251,6 +251,11 @@ boolean forkGitHub(String owner, String repo, String newName, List<String> maint
     private static void setupRepository(GHRepository r, boolean useGHIssues) throws IOException {
         r.enableIssueTracker(useGHIssues);
         r.enableWiki(false);
+        r.setHomepage("https://plugins.jenkins.io/" + r.getName().replace("-plugin", "") + "/");
+        r.setEmailServiceHook("jenkinsci-commits@googlegroups.com");
+        if (!r.getDefaultBranch().equals("main")) {
+            r.setDefaultBranch("main");
+        }
```

Comment:
GitHub recommends creating new repositories with `main` over master, and it's the selected default if you do that through GitHub.
I'd like to build upon that and ensure we don't introduce new degenerative terms.